### ISSUE TITLE
Analyse usage of input pagination

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#10576]
We changed the pagination to work with Next/Prev buttons and an input field. We've had to fix its functioning a few times and we still have three tickets to look at (#10355 #10470 and #10575). We also questioned if that was the best way forward and how accessible it was. 

We'd like to verify how much that input is actually being used and if we shouldn't switch to a different pagination model. 

It might not yet be possible to track this, so the first step of this ticket might be about adding an event to the component.

**Comment by LFBaily** [comment]
Hi @taceybadgerbrook I know we discussed briefly in planning the other day. Think we concluded there wasn't a straightforward way to track usage currently - should we create a ticket for adding event tracking to the input field? Is that possible without a button for a user to interact with? Need to then also consider what usage threshold we would consider significant enough.

I would assume this is most relevant to our researcher audience as most other index/listing pages outside of catalogue and image results don't have particularly high numbers of paginated content (>10)

Re the accessibility issues flagged in #10470 we could explore how we can include a submit button, or consider an alternative solution which may be sufficient to meet user needs eg. adding last and first buttons as well as next and previous, or moving to a pattern where more of the pages can be navigated to directly (eg. 1,2,3,4,5...10).

Also possibly something which could be explored and consolidated as part of a wider piece of work on the design system @j-jaworski ?
